api/views/v1/feedback.py

```diff
@@ -0,0 +1,195 @@
+            The request body can contain:
+                - userID (objectID str) [OPTIONAL]
+                - projectID (objectID str)
+                - taskID (objectID str) [OPTIONAL]
+        @param {str} authorizationToken - JWT authorization token.
+
+        @return A Response object containing a JSON array of the created feedback object.
+
+        @example javascript:
+
+            fetch('quayside.app/api/v1/feedback', {
+                method: 'POST',
+                headers: { 'Content-Type': 'application/json' },
+                body: JSON.stringify({ projectID: 'AF6927BF', taskID: 'AC6927BF' }),
+            });
+
+        """
+        responseData, httpStatus = self.createFeedback(
+            request.data, getAuthorizationToken(request)
+        )
+        return Response(responseData, status=httpStatus)
+
+    def delete(self, request):
+        """
+        Deletes a feedback object or list of feedbacks provided by a user or for a given project.
+        Requires 'apiToken' passed in auth header or cookies to check if the user is an admin.
+
+        @param {HttpRequest} request - The request object.
+            The query parameters MUST be:
+                - id (objectID str)
+                or
+                - projectID (objectID str)
+                or
+                - taskID (objectID str)
+                or
+                - userID (objectID str)
+
+        @return: A Response object with a success or an error message.
+
+        @example javascript:
+
+            fetch(`/api/v1/projects?id=1234`, {
+                method: 'DELETE',
+            });
+        """
+
+        responseData, httpStatus = self.deleteFeedback(
+            request.query_params, getAuthorizationToken(request)
+        )
+        return Response(responseData, status=httpStatus)
+
+    @staticmethod
+    def getFeedback(feedbackData, authorizationToken):
+        """
+        Service API function that can be called internally as well as through the API to get
+        project data based on input data.
+
+        @param feedbackData      Dict for a singular feedback object.
+        @param authorizationToken      JWT authorization token.
+        @return      A tuple of (response_data, http_status).
+        """
+        #TODO: use auth token to check if user is admin
+
+        try:
+            if len(feedbackData) == 1:
+                return { "message": "Need to have ONLY a id, taskID, or projectID property." }, status.HTTP_400_BAD_REQUEST
+            
+            feedback_objs = []
+
+            if "id" in feedbackData:
+                feedback_objs = Feedback.objects.get(id=feedbackData["id"])
+            elif "taskID" in feedbackData:
+                feedback_objs = Feedback.objects.filter(taskID__all=feedbackData["taskID"], **feedbackData)
+            elif "projectID" in feedbackData:
+                feedback_objs = Feedback.objects.filter(projectID__all=feedbackData["projectID"], **feedbackData)
```

Comment:
```suggestion
            if "id" in feedbackData:
                feedback_objs = Feedback.objects.get(id=feedbackData["id"])
            elif "taskID" in feedbackData:
                feedback_objs = Feedback.objects.filter(taskID=feedbackData["taskID"])
            elif "projectID" in feedbackData:
                feedback_objs = Feedback.objects.filter(projectID=feedbackData["projectID"])
```
I did some testing and it looks like the filter `taskID__all=feedbackData["taskID"]` doesn't actually work unless `feedbackData["taskID"]` is a list